AutoYaST/SUSE Multi-Linux Manager Proxy/MLM_RBS-50.xml

```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<!-- Profile updated by dvosburg at suse dot com 20250710 - Upgrade to RBS on SLES 15 SP6
+It assumes system variables plus these entered on the variables tab:
+org='organization_id'
+distrotree='autoinstallation_distribution_label'
+channel_prefix='clm_channel_prefix' or blank for SCC channels
+registration_key='activation_key_for_post_upgrade'
+-->
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+   <add-on>
+    <add_on_products config:type="list">
+<!-- All SLES 15 SP6 repos listed here - comment out what is unneeded -->
+   <!-- Manager Tools repos -->
+      <listentry>
+        <ask_on_error config:type="boolean">true</ask_on_error>
+        #if $channel_prefix != ""
```

Comment:
We probably need a better handling of the missing `channel_prefix`: it fails badly and the only place we have the error is in cobbler logs

Reply:
Currently it needs a variable set in the profile with an empty value.  This was the best way I could handle it at the time - feel free to propose an alternative.